build.sbt

```diff
@@ -56,3 +56,5 @@ lazy val microservice = Project(appName, file("."))
   )
   .settings(PlayKeys.playDefaultPort := 7792)
   .settings(SilencerSettings())
```

Comment:
delete project/SilencerSettings.scala and remove this line since the added `-Wconf:src=routes/.*:silent` setting will do the silencing now